.travis.yml

```diff
@@ -1,28 +1,41 @@
 language: php
 php:
-    - 7.1
-    - 7.2
-    - 7.3
+- 7.1
```

Comment:
Im just curious - why did you remove all those spaces in these arrays?

Reply:
True, please keep the original formatting - it is hard to read without the indentation.

Reply:
Sorry I have no idea how that happend. I will return it to original formatiing